.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+<!--
+Thanks for your interest in contributing to this project. Please follow the Code of Conduct for this project, which can be found in the CODE_OF_CONDUCT.md file.
+
+Fill out this template with information that is relevant to the issue and if necessary provide information about how to replicate the issue so that we can understand the best way to approach the problem.
+-->
+**Problem description**:
+
+**Suggested solution**:
+
+**Relevant code snippet**:
+
+```javascript
+```
+
```

Comment:
Just an idea, but what do you think about adding the following information categories?

1) Steps to Reproduce
2)The current behavior
3) The expected behavior

I stole these from the React open source issue template, linked here if you want to take a look [https://github.com/facebook/react/blob/master/.github/ISSUE_TEMPLATE/bug_report.md](url)

Reply:
+1 this @maxnawa31